formats/definitions.json

```diff
@@ -142,6 +142,51 @@
         }
       }
     },
+    "nodes": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "title"
+        ],
+        "properties": {
+          "kind": {
+            "type": "string"
```

Comment:
If kind has a few valid values, an enum [1] might be better.

1: http://json-schema.org/latest/json-schema-validation.html#rfc.section.5.20